### PR TITLE
Updated markdown of site2cloud resource

### PR DESCRIPTION
**Pull request by zhiqiangjin**
Added algorithms as supported argument, and updated description of some attributes.

### CODE REVIEW

File: aviatrix/resource_site2cloud.go [hunk 1]
```diff
@@ -115,37 +115,31 @@ func resourceAviatrixSite2Cloud() *schema.Resource {
...
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "",
+				Type:     schema.TypeString,
+				Optional: true,
 			},
 			"phase_1_dh_groups": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "",
```
Comment (arvindsree) [inline aviatrix/resource_site2cloud.go]: Add description for these as well

Comment (arvindsree) [reply aviatrix/resource_site2cloud.go]: And to the doc

File: aviatrix/resource_site2cloud.go [hunk 2]
```diff
@@ -115,37 +115,31 @@ func resourceAviatrixSite2Cloud() *schema.Resource {
 			"algorithms_enabled": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Description: "",
+				Description: "IPSec Encryption and Authentication Algorithms.",
```
Comment (arvindsree) [inline aviatrix/resource_site2cloud.go]: This doesnt make sense. It is a boolean.

File: aviatrix/resource_site2cloud.go [hunk 3]
```diff
@@ -115,37 +115,37 @@ func resourceAviatrixSite2Cloud() *schema.Resource {
...
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "",
+				Description: "Phase one Authentication.",
 			},
 			"phase_2_authentication": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "",
+				Description: "Phase two Authentication.",
```
Comment (arvindsree) [inline aviatrix/resource_site2cloud.go]: Provide an example in the doc and list of valid values it accepts for all these parameters. Think of it from the perspective of someone who has seen the UI

File: aviatrix/resource_site2cloud.go [hunk 4]
```diff
@@ -115,37 +115,37 @@ func resourceAviatrixSite2Cloud() *schema.Resource {
 			"algorithms_enabled": {
```
Comment (arvindsree) [inline aviatrix/resource_site2cloud.go]: should be renamed to custom_algorithms

File: aviatrix/resource_site2cloud.go [hunk 5]
```diff
@@ -115,37 +115,37 @@ func resourceAviatrixSite2Cloud() *schema.Resource {
...
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Description: "",
+				Description: "Switch to enable/disable IPSec Encryption and Authentication Algorithms.",
```
Comment (arvindsree) [inline aviatrix/resource_site2cloud.go]: Say "Switch to enable custom/non-default algorithms for IPSec Authenticaton/Encryption"

File: aviatrix/resource_site2cloud.go (hunk 3 above)
Comment (zhiqiangjin) [reply aviatrix/resource_site2cloud.go]: OK.

File: aviatrix/resource_site2cloud.go (hunk 4 above)
Comment (zhiqiangjin) [reply aviatrix/resource_site2cloud.go]: I think custom_algorithms does not like a bool argument, right?

File: aviatrix/resource_site2cloud.go (hunk 5 above)
Comment (zhiqiangjin) [reply aviatrix/resource_site2cloud.go]: Done

File: aviatrix/resource_site2cloud.go (hunk 4 above)
Comment (zhiqiangjin) [reply aviatrix/resource_site2cloud.go]: Done.

**Pull request merged**